### PR TITLE
Added New Play - Do you know the Image?

### LINKED ISSUE #458: [Add a Play]: Do you know the Image?
Thank You! 

## What's the Play about?
This play will look like an image gallery where users can explore many different types of images, and by hovering over it, they can see the video of that place and also listen about that.

https://user-images.githubusercontent.com/91705825/183227342-78745a03-ae60-4083-948b-037f8674ae75.mp4

## What ReactJS concept will be used to create this play?
JSX, useState, map function, and filter funtion.

## Are you willing to take it up for implementation?
Yes!

### PR DESCRIPTION
**Pull request by Shivam-Katare**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

It's an Image Gallery where users can explore multiple types of images and, by hovering over them, they can also listen to that image.

Fixes #458 

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Shivam-Katare** [comment]
Hey, @atapas why the preview of this play is not showing on the vercel deployment?

**Comment by atapas** [comment]
@Shivam-Katare is it ready to start review?

**Comment by Shivam-Katare** [comment]
> @Shivam-Katare is it ready to start review?

Yes, @atapas it's ready.

File: src/plays/image-gallery/Gallery.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import GalleryAPI from './GalleryAPI'
+
+
+export default function Gallery() {
+	const [data, setData] = useState(GalleryAPI)
+	const [ourText, setOurText] = useState("Welcome to this Play, Click on the Images and get to know about it.")
```
Comment (atapas) [inline src/plays/image-gallery/Gallery.js:8]: We can make the text a bit more intutive.

File: src/plays/image-gallery/Gallery.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	const [midText, setMidText] = useState("Sorry for interrupting. I think you like this play. Don't forget to give it a thumbs up.")
+	const msg = new SpeechSynthesisUtterance()
+	var voiceName = "Zira"
```
Comment (atapas) [inline src/plays/image-gallery/Gallery.js:11]: Are there other voice types? Would it be cool to allow users to select one with Zira being the default?

File: src/plays/image-gallery/Gallery.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+	return (
+		<>
+			{/* FInal Grid */}
+
+			<div className='heading-gallery'>Do You Know the Image?</div>
+			<div class="gallery-image"  >
+				{data.map((values) => {
+					const { url, type, discription, title, videoUrl } = values
+					return (
```
Comment (atapas) [inline src/plays/image-gallery/Gallery.js:52]: These indentations are too deep.

File: src/plays/image-gallery/Readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# ImageGallery
+
```
Comment (atapas) [inline src/plays/image-gallery/Readme.md:2]: Add more technical details about the play.

File: src/plays/image-gallery/styles.css [hunk 5]
```diff
@@ -0,0 +1,191 @@
+/* enter stlyes here */
+@import url('https://fonts.googleapis.com/css2?family=Oleo+Script+Swash+Caps&display=swap');
+
```
Comment (atapas) [inline src/plays/image-gallery/styles.css:3]: Like previous plays... append the styles with a unique class name.

**Review by atapas — CHANGES_REQUESTED** [review]
@Shivam-Katare A few initial comments

File: src/plays/image-gallery/Gallery.js (hunk 2 above)
Comment (Shivam-Katare) [reply src/plays/image-gallery/Gallery.js:11]: Means the user can select different voice types?

Comment (atapas) [reply src/plays/image-gallery/Gallery.js:11]: Yes, just a suggestion. We can do it as a version 2 later as well. btw.. let me know when you have taken care of the comments/

@Shivam-Katare

Comment (Shivam-Katare) [reply src/plays/image-gallery/Gallery.js:11]: Okay, then I will resolve all the issues and let you know after that.

**Comment by atapas** [comment]
@Shivam-Katare please let me know when it is ready for a final review.

**Comment by Shivam-Katare** [comment]
> @Shivam-Katare please let me know when it is ready for a final review.

Okay, I will let you know.

**Comment by atapas** [comment]
> > @Shivam-Katare please let me know when it is ready for a final review.
> 
> Okay, I will let you know.

Let's get this in this week?

**Comment by Shivam-Katare** [comment]
> > > @Shivam-Katare please let me know when it is ready for a final review.
> > 
> > 
> > Okay, I will let you know.
> 
> Let's get this in this week?

Yes! @atapas sure. Sorry for the delay; I wasn't able to provide that much time due to exams. But I will complete it this week.

**Review by atapas — APPROVED** [review]
LGTM

**Pull request merged**